lib/github/index.js

```diff
@@ -31,6 +31,7 @@ module.exports = (robot) => {
     'pull_request.reopened',
     'pull_request.edited',
     'pull_request_review',
+    'pull_request.synchronize',
```

Comment:
can you please move this one line up so we can group all `pull_request.X` events together?